joomla/joomla-cms
# [4.0] Cleanup the View files com_banners

### Summary of Changes
This is the next step in cleaning up the view files and adhering to coding standards as well as making the views more usable by the IDE by removing the magic getter.

### Testing Instructions

1. Go to Components -> Banners -> Banners
2. Make sure Banners work as before
3. Go to Components -> Banners -> Clients
4. Make sure Clients work as before
5. Go to Components -> Banners -> Tracks
6. Make sure Tracks work as before

### Documentation Changes Required
None
